src/main/java/edu/cornell/kfs/ksr/document/SecurityGroupMaintainable.java

```diff
@@ -41,5 +47,28 @@ public void processAfterCopy(MaintenanceDocument document, Map<String, String[]>
             }
         }
     }
+    
+    @Override
+    public void doRouteStatusChange(DocumentHeader documentHeader) {
+        super.doRouteStatusChange(documentHeader);
+
+        SecurityGroup securityGroup = (SecurityGroup) getDataObject();
+        createSecurityProvisioningEntryIfItDoesNotExist(securityGroup);
```

Comment:
Although the Cynergy KSR code potentially creates the provisioning object after any route status change, I would suggest updating this "doRouteStatusChange"  override so that it will wait for the document to reach "Processed" status before attempting to create the provisioning object.

Reply:
updated